### PR TITLE
Added explanation and extra info to title

### PR DESCRIPTION
**Pull request by kfklein15**
Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): Week 3

## Description

 Added more information to make sure people understand we are repeating everything weekly. I also added info to the Check In title to make sure people look at it and understand the importance of it.

## Who needs to know about this?

 @MarianaVazquezCYF @martaesn @AliciaE08 @ShaunCYFSA

### CODE REVIEW

File: docs/finalproject/week-3/lesson.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ title: Final Project - Week 3
 sidebar_label: Lesson
 ---
 
-See [Week 2](../week-2/lesson) for instructions for this week
+See [Week 2](../week-2/lesson) for instructions for this week. You are working as a team to deliver a product, so will continue using that format through the Final Projects
 
-### Check In
+### Check In: how is the contribuitions of all team members so far?
```
Comment (SallyMcGrath) [inline docs/finalproject/week-3/lesson.md]:
```suggestion
### Check In: how are the contributions of all team members so far?
```

Minor changes

Subject verb agreement https://www.grammarbook.com/grammar/subjectVerbAgree.asp
Spelling

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Minor changes, but please do use Grammarly to check SPAG before submission. Thanks!

**Comment by kfklein15** [comment]
@SallyMcGrath all accepted and committed.

**Review by SallyMcGrath — APPROVED** [review]
LGTM 🌮

**Pull request merged**